playwright/_impl/_frame.py

```diff
@@ -81,6 +81,9 @@ def __init__(
             lambda params: self._on_frame_navigated(params),
         )
 
+    def __repr__(self) -> str:
+        return f"<Frame name={self.name} url={self.url!r}>"
```

Comment:
Why !r? url is just a string...

Reply:
It would be in quotes and it looks better